docker/transformers-pytorch-gpu/Dockerfile

```diff
@@ -12,7 +12,7 @@ RUN git clone https://github.com/huggingface/transformers && cd transformers &&
 RUN python3 -m pip install --no-cache-dir -e ./transformers[dev-torch,testing,video]
 
 # If set to nothing, will install the latest version
-ARG PYTORCH='2.0.0'
+ARG PYTORCH='2.0.1'
```

Comment:
Let's use the latest version. Otherwise, while building the docker image using this docker file, there will be other version conflicts during build and cause the image being too large and ran out of the disk.